src/components/SelectionsContainer/SelectionsContainer.tsx

```diff
@@ -7,15 +8,17 @@ export interface SelectionsContainerProps {
   buttonLabel: string;
 }
 
-export function SelectionsContainer({
+export const SelectionsContainer = ({
   selection,
   selectionHeading,
   buttonLabel,
-}: SelectionsContainerProps) {
+}: SelectionsContainerProps) => {
   return (
     <div className={`selections ${selection}-section`}>
       <h2 className="selection-heading">{selectionHeading}</h2>
-      <Button label={buttonLabel} />
-    </div>
+      <Link to={`/${selection}`} >
```

Comment:
So glad we figured out how to wrap the button in SelectionsContainer in a Link to route to the correct view and used interpolation to make it dynamic!